example/main.go

```diff
@@ -174,4 +174,8 @@ func main() {
 	if err := run(); err != nil {
 		panic(err)
 	}
+	// try again to run in a new context
+	if err := run(); err != nil {
```

Comment:
This file is an example and we should not add this function just for testing.

Reply:
Okay. Removed